cpp/modmesh/toggle/pymod/wrap_Toggle.cpp

```diff
@@ -151,6 +151,12 @@ WrapProcessInfo::WrapProcessInfo(pybind11::module & mod, char const * pyname, ch
             [](py::object const &) -> auto & {
                 return wrapped_type::instance();
             })
+        .def(
```

Comment:
I did not add a unit test because once set it cannot be redone in the same process.  Testing it requires a clever way that I haven't thought of.